.circleci/dynamic_config.yml

```diff
@@ -136,14 +136,6 @@ commands:
             # cause decryption failures based on the openssl version. https://stackoverflow.com/a/39641378/4317734
             openssl aes-256-cbc -d -in .circleci/gcp_token -md md5 -k "${<< parameters.key >>}" -out /home/circleci/.gcp_credentials;
             sudo bash -c "echo -e 'build --google_credentials=/home/circleci/.gcp_credentials' >> .bazelrc.user";
-            # Upload/don't upload local results to cache based on environment
-            if [[ -n "{$CIRCLE_PULL_REQUEST}" ]]; then
-              sudo bash -c "echo -e 'build:remote --remote_upload_local_results=false\n' >> .bazelrc.user";
```

Comment:
Why was this done? Disabling remote caching for targets where remote execution is disabled...

Normally this would only be done for things like local dev to read but not write to the remote cache. I thought maybe it was to avoid polluting the cache server with random PRs but most bazel targets (all but 3 or 4) get executed remote and are cached despite this flag.

But this is what was causing a few of the unit tests and all the e2e tests (e2e being a [WIP](https://github.com/angular/angular-cli/pull/23074)) to never be cached.